ui/StatusQ/src/StatusQ/Controls/StatusBaseButton.qml

```diff
@@ -80,6 +80,8 @@ Button {
             return 11
         }
     }
+    implicitWidth: content.implicitWidth + 2 * horizontalPadding
+    implicitHeight: content.implicitHeight + 2 * verticalPadding
```

Comment:
Should not be needed at all

Reply:
Does it still work without?

Reply:
Nope! Since the `contentItem` now is a plain `Item` without specific size. 
- If the size is changed from outside like in case of adding the button inside a layout and filling it, then it is resized properly and row content is centered inside the item properly.
- If no size is specified, then it takes the row content item size + paddings! 

Maybe there is a better / elegant way, that's why it's still a draft! Open to suggestions!! :)

Reply:
Yeah I see, good fix in this context. But I'm wondering why it got broken and whether this doesn't conflict with the actual text element:

```qml
StatusBaseText {
  Layout.alignment: root.textAlignment
  Layout.fillWidth: root.textFillWidth
  opacity: !root.loading
  font: root.font
  text: root.text
  color: d.textColor
  elide: Text.ElideRight
}
```

esp. in combination with `root.textFillWidth`

Reply:
Here a better solution: https://github.com/status-im/status-desktop/pull/10543